.github/test-build/action.yml

```diff
@@ -29,26 +29,53 @@ runs:
         npm pack dist/libs/ng-lib --quiet --silent
         npm pack dist/libs/scully-plugin-puppeteer --quiet --silent
         export scullyFolder=$PWD
+        echo "scullyFolder: $scullyFolder"
+    - name: create a new app
+      shell: bash
+      run: |
         cd ..
-        rm ./demo -fr
         echo "Create app with a couple of routes"
         npx  @angular/cli@${{ inputs.angular-version }} new demo --skip-git  --routing --style css --minimal --strict
         cd demo
         npx ng update @angular/cli --migrate-only --from 12.0.0 --to 12.0.1 --allow-dirty
         npx ng g m home --route home --module app.module
         npx ng g m about --route about --module app.module
+    - name: install extra packages (rx, and the Scully using the local packages)
+      shell: bash
+      run: |
+        cd ../demo
         echo "install deps"
         npm i rxjs@7  --legacy-peer-deps --force --quiet --silent
         echo "$scullyFolder"
-        cp $scullyFolder/*.tgz .
-        for f in *.tgz; do
-          echo "install package $f"
-          npm install $f --force --legacy-peer-deps --quiet --silent
+        cp ../scully/*.tgz .
+        for f in  scullyio-scully-2*.tgz; do
```

Comment:
Should the `2` be removed?

Reply:
No, it prevents installing any off the other packages _before_ scully itself